src/components/EditCoursePage/index.jsx

```diff
@@ -480,6 +484,8 @@ class EditCoursePage extends React.Component {
       enterprise_subscription_inclusion,
       organization_short_code_override,
       organization_logo_override_url,
+      value_per_click_usa,
+      value_per_click_international,
```

Comment:
What happens if a non-staff user is editing the page? This can result in setting these values to None.

Reply:
I think it's okay because the default value for these fields is 0.  please let me know if I understand your logic or if I'm still missing something!